dist-git.spec

```diff
@@ -50,7 +50,7 @@ Requires:       python3-requests
 Recommends:     python3-grokmirror
 Suggests:       python3-fedmsg
 Suggests:       fedora-messaging
-BuildRequires:  python3-nose
+BuildRequires:  python3-pytest
```

Comment:
Looks good, but please give me some time to test this.  Random mock build failed for me...
I think we should setup Packit here in this repo first.